chainercv/links/model/ssd/multibox_coder.py

```diff
@@ -0,0 +1,238 @@
+from __future__ import division
+
+import itertools
+import numpy as np
+
+import chainer
+
+from chainercv import utils
+
+
+class MultiboxCoder(object):
+    """A helper class to encode/decode bounding boxes.
+
+    This class encodes :obj:`(bbox, label)` to :obj:`(mb_loc, mb_label)`
+    and decodes :obj:`(mb_loc, mb_conf)` to `(bbox, label, score)`.
+    These encoding/decoding are used in Single Shot Multibox Detector [#]_.
+
+    .. [#] Wei Liu, Dragomir Anguelov, Dumitru Erhan, Christian Szegedy,
+       Scott Reed, Cheng-Yang Fu, Alexander C. Berg.
+       SSD: Single Shot MultiBox Detector. ECCV 2016.
```

Comment:
It would be nice if you have a description on what `bbox`, `mb_loc`, `mb_loc` and `mb_label`.
The relationship between `bbox` and `mb_loc` can be described using formulas as done in `bbox2loc` and `loc2bbox`.